### PR TITLE
[Tests-Only] Visually check all tabs

### PR DESCRIPTION
**Pull request by swoichha**
## Description
This PR adds test to check if all the tabs in toolbar section is visible by property verification point creation and suing squish api `test.vp` 

## Related Issue
 - https://github.com/owncloud/client/issues/8705

Also see: https://github.com/owncloud/client/issues/8651#issuecomment-857373491

### CODE REVIEW

File: test/gui/tst_checkAlltabs/verificationPoints/Activity [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="pos.x" type="int" valueType="Value"><![CDATA[195]]>
+<base64EncodedValue>MTk1</base64EncodedValue>
+</Verification>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="pos.y" type="int" valueType="Value"><![CDATA[0]]>
+<base64EncodedValue>MA==</base64EncodedValue>
+</Verification>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="text" type="QString" valueType="Value"><![CDATA[Activity]]>
+<base64EncodedValue>QWN0aXZpdHk=</base64EncodedValue>
+</Verification>
+<Verification object="settings_settingsdialog_toolbutton_Activity_QToolButton" property="visible" type="bool" valueType="Value"><![CDATA[true]]>
```
Comment (Talank) [inline test/gui/tst_checkAlltabs/verificationPoints/Activity:13]: is there any way to get these objects from page objects instead of `names.py`?  🤔

Comment (swoichha) [reply test/gui/tst_checkAlltabs/verificationPoints/Activity:13]: I don't think we can do that. They are auto generated when recording steps using verification point just like it is done here https://github.com/owncloud/client/blob/5211ff80696190e635fd9f7d77ddaf0074aaa569/test/gui/shared/verificationPoints/publicLinkExpirationProgressIndicatorInvisible#L4

**Review by Talank — COMMENTED** [review]
LGTM

File: test/gui/shared/steps/steps.py [hunk 2]
```diff
@@ -640,3 +640,11 @@ def step(context):
         'Ser&ver Address',
     )
     waitForObject(AccountConnectionWizard.SERVER_ADDRESS_BOX)
+
+
+@Then("all tabs in the toolbar should match the default baseline")
```
Comment (individual-it) [inline test/gui/shared/steps/steps.py:645]: could we make the tab names as parameter? So that every can be tested by itself

**Review by individual-it — CHANGES_REQUESTED** [review]
CI fails with some linting issue, not sure if its related

**Comment by swoichha** [comment]
> CI fails with some linting issue, not sure if its related

It has nothing to so with this PR. `test-clang-format` has been failing in CI for 2 days now.

File: test/gui/shared/steps/steps.py (hunk 2 above)
Comment (swoichha) [reply test/gui/shared/steps/steps.py]: Done

File: test/gui/shared/steps/steps.py [hunk 3]
```diff
@@ -640,3 +640,11 @@ def step(context):
         'Ser&ver Address',
     )
     waitForObject(AccountConnectionWizard.SERVER_ADDRESS_BOX)
+
+
+@Then("the following tabs in the toolbar should match the default baseline")
+def step(context):
+
+    for tabName in context.table:
+        print(tabName)
```
Comment (individual-it) [inline test/gui/shared/steps/steps.py:649]: do we need this print?

**Review by individual-it — CHANGES_REQUESTED** [review]

File: test/gui/shared/steps/steps.py (hunk 3 above)
Comment (swoichha) [reply test/gui/shared/steps/steps.py]: No we don't need that it was added just for debug purpose. Now it's removed.

**Review by individual-it — APPROVED** [review]
please squash on merge

**Pull request merged**